extensions/calDotCom/actions/getBooking.ts

```diff
@@ -90,7 +90,7 @@ export const getBooking: Action<typeof fields, typeof settings> = {
           status: booking.status,
           cancelUrl: `https://app.cal.com/booking/${booking.uid}?cancel=true`,
           rescheduleUrl: `https://app.cal.com/reschedule/${booking.uid}`,
-          videoCallUrl: booking.metadata.videoCallUrl ?? '',
+          videoCallUrl: booking.metadata.videoCallUrl,
```

Comment:
@kubiak85 sorry, my comment wasn't asking for a change. It was just to make a note for future reference -- please revert this

Reply:
The 'requested changes' was to resolve the merge conflicts